docs/README.md

```diff
@@ -192,6 +192,17 @@ Avoid.
 See [sets](#sets).
 ([#727](https://github.com/web-platform-dx/web-features/pull/727#discussion_r1537635491))
 
+##### typically styled as … by default
+
+Use this phrase (or a closely similarly one like "typically styled in") to note unspecified but conventional browser default styling.
```

Comment:
The default styling is actually specified in detail and quite interoperable. For `<small>`, it's here:

https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3

I think I'd prefer us to say just "by default" for these cases.

Reply:
Oh fascinating! That's worth amending over. Changes forthcoming